packages/typescript-sdk/src/generated/base/requests-utils.ts

```diff
@@ -64,36 +65,39 @@ function reduceMiddleware(op1: Middleware, op2: Middleware): Middleware {
   }
 }
 
+function isDefined<T>(value: T | undefined): value is T {
+  return typeof value !== "undefined"
```

Comment:
i would add the null check here
```suggestion 
  return typeof value !== "undefined" && value !== null
```

Reply:
Fixed, thanks. I wasn't sure about this since null would in almost all cases be explicitly set.